src/bci_build/package/appcontainers.py

```diff
@@ -278,8 +278,11 @@ def _get_nginx_kwargs(os_version: OsVersion):
         "build_recipe_type": BuildType.DOCKER,
         "extra_files": _NGINX_FILES,
         "support_level": SupportLevel.L3,
-        "exposes_ports": [TCP(80)],
+        "exposes_ports": [TCP(8080)],
         "custom_end": f"""{version_check_lines}
+# Modify the default Nginx config to listen on port 8080 and PID file location to a writable path
+{DOCKERFILE_RUN} sed -i 's/listen  *80;/listen 8080;/g' /etc/nginx/nginx.conf &&\\
```

Comment:
This is a breaking change for anyone using the container as is.

In my view, if you want to change the container port and user, you must provide your own nginx.conf file.

I would just downgrade the nginx process by setting the worker user to nginx.